TomTomSDKExamples/Examples/UseCase/Offline/OfflineNavigationContent.swift

```diff
@@ -0,0 +1,689 @@
+        let updateConfig = NDSStoreUpdateConfig(
+            updateStoragePath: updateStoragePath,
+            persistentStoragePath: persistantStoragePath,
+            updateServerURL: updateServerURL,
+            updateServerAPIKey: Keys.apiKey,
+            iqMapsAllRegionsEnabled: false,
+            iqMapsRelevantRegionsEnabled: true,
+            iqMapsRelevantRegionsRadius: .tt.kilometers(10),
+            iqMapsRelevantRegionsUpdateInterval: .tt.minutes(60),
+            iqMapsRegionsAlongRouteEnabled: true,
+            iqMapsRegionsAlongRouteRadius: .tt.kilometers(5)
+        )
+
+        /// Create the "NDSStore" object by the given configurations
+        guard let ndsStore = NDSStore(
+            mapDataPath: mapDataPath,
+            keystorePath: keystorePath,
+            accessPermit: .mapLicense(Keys.mapLicense),
+            ndsStoreUpdateConfig: updateConfig
+        ) else {
+            /* YOUR CODE GOES HERE */
+            /// Check your map license and api key
+            fatalError("Error!")
+        }
+
+        /// It is important to note that the relevant region updates have been enabled above through the use of the "NDSStoreUpdateConfig" configuration.
+        /// Enabling updates and setting the position of the "NDSStore" object will automatically update the relevant regions of the position.
+        /// As a result, in the case below, after a certain amount of time (depending on internet connection) the Amsterdam region will be downloaded and displayed on the map.
+        /// Once downloaded, the following lines can be removed and the app can be rerun. The previously downloaded regions will still be available on the device.
+        /// Additionally, downloading for any other coordinate can also be attempted.
+        ndsStore.updatesEnabled = true
+        ndsStore.setPosition(CLLocationCoordinate2D(
+            latitude: Region.amsterdam.lat,
+            longitude: Region.amsterdam.lon
+        ))
+
+        /// Create an offline data provider
+        var dataProviders: [MapDisplayDataProvider] = []
+        if let offlineDataProvider = OfflineTileDataProviderFactory.createOfflineTileDataProvider(store: ndsStore) {
+            dataProviders.append(offlineDataProvider)
+        } else {
+            /* YOUR CODE GOES HERE */
+            fatalError("Error!")
+        }
+
+        /// Create the map options by creted data provider and other configurations
+        let mapOptions = MapOptions(
+            apiKey: Keys.apiKey,
+            cachePolicy: .noCaching,
+            styleMode: .main,
+            dataProviders: dataProviders
+        )
+
+        /// Create the "MapView" object with the map options
+        mapView = TomTomSDKMapDisplay.MapView(mapOptions: mapOptions)
+    }
+}
+
+// MARK: UIViewRepresentable
+
+/// Extend the TomTomOfflineMapView to conform to the UIViewRepresentable protocol.
```

Comment:
Not really useful comment since it duplicates the definition, and doesn't explain it. having one below is enough.

Reply:
👍 Done!